tiled/client/xarray.py

```diff
@@ -2,6 +2,7 @@
 import builtins
 import itertools
 
+import pandas
```

Comment:
[To recap a Zoom converstaion] Xarray depends on it anyway, but you are right that we should explicitly list it now since we import it directly. That way, we are future-proofed against xarray possibly dropping it as a required dependency.